Pipfile

```diff
@@ -13,7 +13,7 @@ django-haystack = "*"
 "boto3" = ">=1.9.16"
 botocore = '>=1.12.16'
 requests = "*"
-Django = ">=2.1.7"
+Django = "===2.1.7"
```

Comment:
Should this be `~=2.1.8`?

Reply:
I'm resetting the pipfile for this branch... there's no reason for it to be different from master

Reply:
Ah, good